.github/workflows/validate_packs.yml

```diff
@@ -26,7 +26,7 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v2
         with:
-          ref: 'refs/pull/${{ steps.get_pr_number.outputs.pr-number }}/merge'
+          ref: "refs/pull/${{ steps.get_pr_number.outputs.pr-number }}/merge"
```

Comment:
i dont think you need any changes to this file.

your other pr has these changes, but also nothing is added here just formatting changes.

Reply:
It's the prettier extension. I've converted it back to single quotes but it also updated line 5 which was originally in double quotes. I'm not sure how to enable it so it allows both

Reply:
id say just dont commit changes to this file in the branch. you can checkout the version of the file on main commit that (without saving), and that should just reset it.